speech/speech.bas

```diff
@@ -1,10 +1,17 @@
+#INCLUDE "../common/welcome.inc"
+
 Intro$="Welcome to the Colour Maximite 2"
 Prompt$="Please enter something for me to speak, or Q to exit"
 
 PhoneticOn=0
 Speed=72
 Pitch=64
 
+MODE 3,8
+CLS
+we.clear_keyboard_buffer()
+
+

```

Comment:
This below:
`IF` Answer$="Q" OR Answer$="q"
can be replaces by:
`IF we.is_quit_key%(Answer$)`